ucad.c

```diff
@@ -242,7 +244,7 @@ ucad_zmq_create_image_header (gpointer buffer, guint width, guint height, guint
     gchar *timestamp;
 
     if (builder == NULL) {
-        builder = json_builder_new_immutable ();
+        builder = json_builder_new_immutable();
```

Comment:
oops, formatting change and we use space everywhere else, so please change back.